content/js1/sprints/4/prep/index.md

```diff
@@ -8,4 +8,666 @@ backlog= 'Module-JS1'
 backlog_filter= 'Week 4'
 +++
 
+## Pre-requisites
 
+## Learning objectives
+
+- Execute a test script
+- Isolate a test case in a test suite
+- Differentiate between running code with `node` command and with an `npm` script, such as `npm test`
+- Interpret documentation to determine how part of an 3rd API behaves
+- Define the term _API_
+- Identify the unit under test in a given test case
+- Outline the effect of running `npm test`
+- Define _dependency_
+- Explain why we use interfaces in programming
+- List the benefits of using a 3rd party software library when programming
+- Explain the purpose of each syntactic element of a test case implemented in Jest
+- Implement a test case to describe the behaviour of a function
+- Outline the effects of running an installation command, e.g. `npm install`
+- Describe what `toBe` checks in the Jest library
+- Suggest potential Jest matchers to check a specified behaviour for a function
```

Comment:
I think we'd probably need to cover using at least one more matcher to call this covered.